server/src/Api/Controllers/GameController.cs

```diff
@@ -22,4 +27,22 @@ public Task<int> CalculateScore(CalculateScoreCommand command)
     {
         return _mediator.Send(command);
     }
+    
+    [HttpPost(Routes.Game.Create)]
+    public Task<Result> CreateGame(string gameName)
+    {
+        return _mediator.Send(new CreateGameCommand(gameName));
+    }
+    
+    [HttpGet(Routes.Game.AllGames)]
+    public Task<IEnumerable<Game>> GetAllGames()
+    {
+        return _mediator.Send(new GetAllGamesQuery());
+    }
+    
+    [HttpGet(Routes.Game.GameById)]
+    public Task<Game> GetGameById(Guid gameId)
```

Comment:
GetById is a more technical term, pls rename to `GetDetails` also rename endpoint `details/{gameId}`